queries/tera/highlights.scm

```diff
@@ -0,0 +1,158 @@
+; Variables
+;----------
+(identifier) @variable
+
+((identifier) @variable.builtin
+  (#any-of? @variable.builtin "loop" "__tera_context"))
+
+; Properties
+;-----------
+(member_expression
+  property: (identifier)? @variable.member)
+
+; Literals
+;-----------
+(string) @string
+
+(bool) @constant.builtin
+
+(number) @number
+
+; Tokens
+;-----------
+[
+  "."
+  ","
+] @punctuation.delimiter
+
+[
+  "*"
+  "/"
+  "%"
+  "|"
+  "+"
+  "-"
+  "~"
+  "="
+  "=="
+  "!="
+  "<"
+  ">"
+  "<="
+  ">="
+] @operator
+
+[
+  "("
+  ")"
+  "["
+  "]"
+  "{%"
+  "%}"
+  "-%}"
+  "{%-"
+  "}}"
+  "{{"
+  "-}}"
+  "{{-"
+  "::"
+] @punctuation.bracket
+
+; Tags
+;-----------
+(comment_tag) @comment
+
+; Keywords
+;-----------
+[
+  "if"
+  "elif"
+  "else"
+  "endif"
+] @keyword.control.conditional
```

Comment:
You're using Helix captures -- we don't have `@keyword.control` but capture `@keyword.conditional` and friends directly, see https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#keywords